src/index.html

```diff
@@ -3,12 +3,53 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#Home">
+        <img class="logo"
+        src="images/logo.png"
+        alt="Moyologo">
```

Comment:
fix the code style
![image](https://user-images.githubusercontent.com/22824431/236446284-d7bbb8f1-162f-4259-97e2-9350c117fd6a.png)
```suggestion
        <img
          class="logo"
          src="images/logo.png"
          alt="Moyologo"
        >
```